packages/asset-swapper/src/utils/market_operation_utils/index.ts

```diff
@@ -578,10 +580,13 @@ export class MarketOperationUtils {
             }
         }
         const collapsedPath = optimalPath.collapse(orderOpts);
+
         return {
             optimizedOrders: collapsedPath.orders,
             liquidityDelivered: collapsedPath.collapsedFills as CollapsedFill[],
             sourceFlags: collapsedPath.sourceFlags,
+            adjustedRate: collapsedPath.adjustedRate(),
+            exchangeProxyOverhead: optimizerOpts.exchangeProxyOverhead,
```

Comment:
Why do you want to return `exchangeProxyOverhead`? can't you simply fetch it from `_opts`?

Reply:
(L599)

Reply:
discussed